### PR TITLE
Update `.travis.yml` and up min. suggested AiiDA Lab version

### LINKED ISSUE #5: .travis.yml file should (perhaps) be changed for "new" app / testing apps
I think the current content of the `.travis.yml` file is not correct.
I believe it is taken from an existing app, but I am not sure.

First, it tries to install the app, including `precommit` and `testing` packages, however, these do not exist for the current app.
In the end, it is probably fine if these are simply removed?
Perhaps some guide-lines should be written concerning how to test AiiDA Lab Apps as well?

Do we have a common or "normal" way of testing apps? - Making sure the AiiDA db env is loaded, how to test Jupyter notebooks, etc?

### LINKED ISSUE #8: Up minimum AiiDA Lab version
The suggested minimum AiiDA Lab version should be upped to the newest minor release version, which is used in the newest [Quantum Mobile](https://github.com/marvel-nccr/quantum-mobile/releases/): 19.3.0

### PR DESCRIPTION
**Pull request by CasperWA**
Remove `precommit` and `testing` packages from the new app's `.travis.yml` file, since they do not exist in the app.

Fixes #5.

Further updates should come according to the discussion in issue #6.

---

Upped minimum suggest AiiDA Lab version to 19.3.0.

Fixes #8.

### CODE REVIEW

File: {{cookiecutter.repo_name}}/.travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ addons:
 install:
 # Upgrade pip setuptools and wheel
 - pip install -U pip wheel setuptools
-- pip install --no-use-pep517 .[precommit,testing]
+- pip install .
```
Comment (ltalirz) [inline {{cookiecutter.repo_name}}/.travis.yml]: the app doesn't have a `setup.py`, so does this do anything?
(i.e. shouldn't it be  `pip install -r requirements.txt`?)

Can I suggest you add this step also in the top-level .travis.yml, similar to what is done in the plugin cutter https://github.com/aiidateam/aiida-plugin-cutter/blob/master/.travis.yml#L59 ?

Comment (CasperWA) [reply {{cookiecutter.repo_name}}/.travis.yml]: Done.

The update to the top-level `.travis.yml` file can be found in the diff.

The main thing is now that a pre-commit is done for the cookiecutter-created repo.
This also requires the install of aiidalab, which makes the tests last much longer than previously.

**Review by ltalirz — APPROVED** [review]
looks good!

**Comment by CasperWA** [comment]
> looks good!

Thanks - I'll just up the suggested minimum AiiDA Lab version here as well (in an additional commit) and close the newly created issue as a bonus.

**Review by ltalirz — APPROVED** [review]
(y)

**Pull request merged**